caravel/assets/javascripts/explorev2/components/FieldSet.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PropTypes } from 'react';
```

Comment:
this component is called `FieldSet`, but really appears to render a single field. Maybe renaming it to `Field`, or `FieldFactory`